packages/node/src/generators/setup-docker/files/Dockerfile__tmpl__

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/node:lts-alpine
+
+WORKDIR <%= app %>
```

Comment:
Let's just make this `WORKDIR /app` to avoid the verbosity. It doesn't matter if it matches the host machine's source root or not.